pkg/util/flag/flags.go

```diff
@@ -22,18 +22,10 @@ import (
 	"strconv"
 
 	"github.com/spf13/pflag"
-	"k8s.io/klog"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
 )
 
-// PrintFlags logs the flags in the flagset
-func PrintFlags(flags *pflag.FlagSet) {
```

Comment:
This seems like an exported function that could have a lot of external consumers. Wondering if we should leave this copy here to be as less disruptive to those consumers but add a comment pointing to the new preferred location.

Reply:
Is there a way to verify if there are external consumers for this function?

Reply:
Is that possible to handle this concern by adding the switch to some documentation and maybe leave comments in code?

Reply:
I'll defer to @liggitt on this one. I don't think it hurts to leave the extra copy here with comments that link to the new location but maybe there aren't as many consumers of this than I initially thoughts.

Reply:
I think people shouldn't be directly importing this package. We discourage importing things not published in one of the staging repos.